### PR TITLE
Update site with 2021 roadmap

### PR DESCRIPTION
**Pull request by Callisto13**
### Description

Update the community page (Remember that existed? I sure didn't) with our 2021 roadmap goals.

([Sort of rendered](https://github.com/Callisto13/eksctl/blob/community-roadmap/userdocs/src/community.md)).

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: userdocs/src/community.md [hunk 1]
```diff
@@ -7,20 +12,68 @@
...
 
-### Declarative configuration management for clusters
+The first stage of this is implementing support for [Flux v2](https://fluxcd.io/).
+This support already exists in an experimental form in `eksctl` (`eksctl enable flux`)
+and work there will carry on throughout the year.
+The next stage is to deprecate support for Flux v1, and expand the capacity of
+quickstart profiles. More information can be found [here](https://github.com/weaveworks/eksctl/issues/2963).
 
-One should be able to make EKS cluster configuration through declarative config files (`eksctl apply`). Additionally, they should be able to manage a cluster via a git repo.
+### Terraform
```
Comment (michaelbeaumont) [inline userdocs/src/community.md:72]:
```suggestion
### Terraform integration
```

File: userdocs/src/community.md [hunk 2]
```diff
@@ -7,20 +12,68 @@
...
 
-### Cluster addons
+This is another in-demand feature which has finally made it to the list.
+Progress can be followed [here](https://github.com/weaveworks/eksctl/issues/1094).
 
-Understanding how the [add-ons spec by SIG Cluster Lifecycle](https://github.com/kubernetes/enhancements/pull/746) will evolve and how we can implement management of some cluster applications via the addon spec.
+An `eksctl` Terraform provider will let users manage their AWS clusters in the
+same format as for their other providers, without needing to understand the intricacies
+of EKS directly.
```
Comment (michaelbeaumont) [inline userdocs/src/community.md:79]: Could be less definite? I.e. at the moment a provider is still just one of the options, right?

```suggestion
One option would be support for an `eksctl` Terraform provider that would let users manage their AWS clusters in the
same format as for their other providers, without needing to understand the intricacies of EKS directly.
```

Comment (Callisto13) [reply userdocs/src/community.md:79]: If we are still deciding on what we want to do wrt terraform, then i feel we probably should not mention it here but just have it in the researching section of the roadmap

Comment (michaelbeaumont) [reply userdocs/src/community.md:79]: Ok, then my stance is don't mention it on this page

File: userdocs/src/community.md [hunk 3]
```diff
@@ -7,20 +12,60 @@
...
+
+## 2021 Roadmap
+
+The following are the features/epics we will focus on and hope to ship this year.
+We will take their completion as a marker for graduation to v1.
+General maintenance of `eksctl` is still implied alongside this work,
+but all subsequent features which are suggested during the year will be weighed
+in relation to the core targets.
+
+Progress on the roadmap can be tracked [here](https://github.com/weaveworks/eksctl/projects/2).
```
Comment (aclevername) [inline userdocs/src/community.md:29]: shall we add the issues to the project before updating the site?

Comment (Callisto13) [reply userdocs/src/community.md:29]: i don't think one blocks the other, but is on my list 👍

**Review by aclevername — APPROVED** [review]
LGTM

**Pull request merged**